homeassistant/components/pooldose/diagnostics.py

```diff
@@ -0,0 +1,50 @@
+"""Diagnostics support for Pooldose."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.core import HomeAssistant
+
+from . import PooldoseConfigEntry
+
+TO_REDACT = {
+    "IP",
+    "MAC",
+    "WIFI_SSID",
+    "AP_SSID",
+    "SERIAL_NUMBER",
+    "DEVICE_ID",
+    "OWNERID",
+    "NAME",
+    "GROUPNAME",
+}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: PooldoseConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    coordinator = entry.runtime_data
+
+    return {
+        "entry": {
+            "title": entry.title,
+            "unique_id": entry.unique_id,
```

Comment:
isnt the unique id the serial number?

Reply:
removed the whole entry block as the only interesting information is the unique_id, but this contains the s/n - to be reducted.

Reply:
fixed in 3015b81a338875fb651f33bf30fbb69cab716350